.github/workflows/build-windows.yml

```diff
@@ -266,46 +266,3 @@ jobs:
           name: windows${{inputs.build-suffix}}-installer-signed
           path: |
             lantern-installer${{inputs.installer-suffix}}.exe
-
-  upload-windows:
```

Comment:
@atavism Do we need upload-windows here?

Reply:
Do we not plan to upload Windows builds to S3 anymore?

Reply:
Hey this is my bad.  I gonna close this.